plugin/gcp_account_resources.go

```diff
@@ -128,7 +147,147 @@ func (b *backend) createServiceAccount(ctx context.Context, req *logical.Request
 		return nil, err
 	}
 
-	return iamAdmin.Projects.ServiceAccounts.Create(fmt.Sprintf("projects/%s", project), createSaReq).Do()
+	return iamAdmin.Projects.ServiceAccounts.Create(fmt.Sprintf("projects/%s", project), createSaReq).Context(ctx).Do()
+}
+
+// tryDeleteGcpAccountResources creates WALs to clean up a service account's
+// bindings, key, and account (if removeServiceAccount is true)
+func (b *backend) tryDeleteGcpAccountResources(ctx context.Context, req *logical.Request, boundResources *gcpAccountResources, removeServiceAccount bool, walIds []string) []string {
+	if boundResources == nil {
+		b.Logger().Debug("skip deletion for nil roleset resources")
+		return nil
+	}
+
+	b.Logger().Debug("try to delete GCP account resources", "bound resources", boundResources, "remove service account", removeServiceAccount)
```

Comment:
```suggestion
	b.Logger().Debug("try to delete GCP account resources", "bound_resources", boundResources, "remove_service_account", removeServiceAccount)
```

The keys on logger tend to be snake cased.